### PR TITLE
Add outputs command

### PR DESCRIPTION
**Pull request by jantman**
This adds `outputs` and `outputs_json` Rake tasks to show the terraform outputs.

### CODE REVIEW

**Comment by dkolb** [comment]
From my closed pull request: You'll need to add the `-auto-approve` flag to any calls to `apply`.

See first item in the [0.10.0 changelog](https://github.com/hashicorp/terraform/blob/master/CHANGELOG.md#0100-august-2-2017).

This will require updating quite a few specs with the new argument and also will break compatibility with 0.9 terraform, making tfwrapper 0.2.0 the final version that supports terraform 0.9.

**Comment by jantman** [comment]
@dkolb Thanks so much for the work and looking into this, and for making this work with future versions before it breaks.

No need to break compatibility; we're already [parsing the output](https://github.com/manheim/tfwrapper/blob/ee4ae321ada79035f8d7ead6e9eaf0ee5c14411a/lib/tfwrapper/raketasks.rb#L283) of `terraform -version` into a `Gem::Version` to check compatibility; the commits that I just pushed save that version as an instance variable and use it to conditionally add the `-auto-approve` arg if >= 0.10.0.

I have one more change I want to make to this branch, but after that, it would be great if I could get a code review from you and/or someone on tooling...

**Comment by jantman** [comment]
@dkolb @jniesen @reppard @duckpuppy Does one of you have a few minutes to look over and approve this PR?

File: ChangeLog.md
Comment (duckpuppy) [inline ChangeLog.md]: Being picky... it's not 0.9.0, it's 0.9.2

Comment (duckpuppy) [reply ChangeLog.md]: Otherwise, I see no issues.

**Review by duckpuppy — APPROVED** [review]

**Pull request merged**
